File: FCUBEAPI/FreightMasters/Models/AddCostRecListModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FreightMasters.Models
{
    public class AddCostRecListModel
    {
        public List<AddCostRecMstModel> AddCostRecMstList { get; set; }
        public PaginationMetaData PageMetaData { get; set; }
    }
}

File: OMKARAUTOAPP/src/app/models/addcostreclistmodel.ts
import { Paginationmodel } from "./paginationmodel";
import { Addcostrecmstmodel } from "./addcostrecmstmodel";

export class Addcostreclistmodel {
    addCostRecMstList: Addcostrecmstmodel[] = [];
    pageMetaData: Paginationmodel = new Paginationmodel;
}
